SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cff.py

```diff
@@ -45,7 +45,10 @@
 from Configuration.Eras.Era_Run3_2023_UPC_cff import Run3_2023_UPC
 (pp_on_PbPb_run3_2023 | Run3_2023_UPC).toModify(HcalTPGCoderULUT, FG_HF_thresholds = [16, 19])
 
-#placedholder values for 2024, copied from 2023
 from Configuration.Eras.Modifier_pp_on_PbPb_run3_2024_cff import pp_on_PbPb_run3_2024
 from Configuration.Eras.Era_Run3_2024_UPC_cff import Run3_2024_UPC
 (pp_on_PbPb_run3_2024 | Run3_2024_UPC).toModify(HcalTPGCoderULUT, FG_HF_thresholds = [16, 19])
+
+#placedholder values for 2025, copied from 2024
+from Configuration.Eras.Modifier_run3_oxygen_cff import run3_oxygen
+run3_oxygen.toModify(HcalTPGCoderULUT, FG_HF_thresholds = [16, 19])
```

Comment:
Should we not also modify Era_Run3_2025_UPC_cff and grab and modify Modifier_pp_on_PbPb_run3_2025_cff to be consistent with the original PR?: https://github.com/stahlleiton/cmssw/blob/e3fec58adb20e683bd0d0da9a563e91d34481543/SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cff.py#L52-L56

Reply:
I had to remove that modifier because it was not present in 15_0_X

Reply:
`Era_Run3_2025_UPC_cff` is available. And do you not want to include `Modifier_pp_on_PbPb_run3_2025_cff` and then modify it?

Reply:
Well, since this modifier (pp_on_PbPb_run3_2025) will only be relevant for the PbPb run at the end of this year, which probably will be done on 15_1_X or later, I thought it was not worth it to back propagate that modifier. But maybe @mandrenguyen can clarify

Reply:
@BenjaminRS : I updated the backport to also include the missing changes to be fully consistent with the original PR.

Reply:
Yes, I would just backport to 15_0_X to keep things uniform, unless there's a good reason not to.